### PR TITLE
Updated API versions from v58.0 to v59.0

### PR DESCRIPTION
**Pull request by groundwired**
# Critical Changes
RollupDmlGuard API version updated to v58.0

# Changes
No code changes - just version updates in two files that were missed

# Issues Closed
https://github.com/SFDO-Community/declarative-lookup-rollup-summaries/issues/1396

### CODE REVIEW

File: config/shape-scratch-def.json
Comment (aheber) [inline config/shape-scratch-def.json:1]: I don't think this file should be included in the PR, can you please remove it or justify it?

**Comment by aheber** [comment]
This is great but it is actually time to bring everything up to API V59.0, would you be willing to change this to V59.0?

If you can cleanup the extra file and sign the CLA that should be everything we need to accept this! Really appreciate you bringing a problem and a solution.

**Comment by groundwired** [comment]
> This is great but it is actually time to bring everything up to API V59.0, would you be willing to change this to V59.0?
> 
> If you can cleanup the extra file and sign the CLA that should be everything we need to accept this! Really appreciate you bringing a problem and a solution.

Should be all set!

**Comment by aheber** [comment]
This looks great, thank you for expanding it! You even caught all the non-xml versions that most people miss.

I'll have to figure out what we need to do to get the automated tests to run so we can accept this. I think we might need to recreate the branch inside the team-scope but I'll get back to you on that.

**Comment by groundwired** [comment]
> This looks great, thank you for expanding it! You even caught all the non-xml versions that most people miss.
> 
> I'll have to figure out what we need to do to get the automated tests to run so we can accept this. I think we might need to recreate the branch inside the team-scope but I'll get back to you on that.

That seems wise!  FWIW, I did run all Apex tests in the package in my scratch org. Everything passes.

**Comment by aheber** [comment]
@groundwired I think you should have received an invite to join the project. That should allow you to close this PR and your branch under your name. Push these changes to a branch directly on the project an open a new PR. That will allow the automation to run and validate your changes and get the needed passes so we can merge.

Sorry for the trouble but hopefully it isn't a lot of work.

**Pull request closed without merging**